### ISSUE TITLE
10.8 RC1 doesn't preserve key mappings or screen dimensions

### ISSUE THREAD

**Issue by Dadtheimpaler** [vpinball/vpinball#1456]
**Describe the bug**
When launching 10.8 RC1, key mappings are defaulted, as are screen measurements.
Mappings exist in the vpinballx.ini, but are not used (or the wrong ones are being used).
Launching older 10.8 versions makes use of the mappings and measurements properly.
10.8 RC1 creates some duplicate mappings in vpinballx.ini, such as joyrflipkey and JoyRFlipKey - is this expected?

**To Reproduce**
1. Launch an older beta version of 10.8 and note button assignments.
2. Launch 10.8 RC1 and note button assignments.
3. Compare vpinballx.ini after launching/losing each version.

**Expected behavior**
Button assignments should be the same
Would not expect duplicate mappings to exist 

**Screenshots**
I've attached a screenshot that shows the vpinballx.ini after launching 10.8 RC1, and after launching an older 10.8 beta.
Note that there are some duplicated values - 

**Crash dump**
n/a

**Versions & Settings**
 - VPX: 10.8.0 1544 and 10.8 RC1 1873 (64-bit in both cases)
 - Windows 11
 - AMD RX 6800
 - vpinballx.ini files attached
 - If this bug report is for VPX 10.8+, please enable log in your settingd and join your log file - How???

**Additional context**

I was able to workaround this by editing the .ini file so that, for instance, joyrflipkey and JoyRFlipKey had the same values
Totally unrelated, but fwiw: I've also noticed that RC1 plays tables at 60fps, despite being set for 120Hz in VPX. If I then launch 10.8 1544, tables play at 120fps.

[VPinballX - RC1.log](https://github.com/vpinball/vpinball/files/14255674/VPinballX.-.RC1.log)
[VPinballX - 1544.log](https://github.com/vpinball/vpinball/files/14255675/VPinballX.-.1544.log)

![VPX 10 8 RC1 ini diff](https://github.com/vpinball/vpinball/assets/7451004/b38f256c-548f-45ba-baab-32f98e210062)

**Comment by francisdb** [comment]
In 10.8 the settings file is case sensitive. The correct keys can be found at https://github.com/vpinball/vpinball/blob/master/src/assets/Default%20VPinballX.ini

As for the fps. Maybe your sync mode settings also are not found because of the wrong casing? Causing it to default to frame pacing at your monitor 's refresh rate?

**Comment by Dadtheimpaler** [comment]
> In recent 10.8 versions the settings file is case sensitive. The correct keys can be found at https://github.com/vpinball/vpinball/blob/master/src/assets/Default%20VPinballX.ini
> 
> As for the fps. Maybe your sync mode settings also are not found because of the wrong casing? Causing it to default to frame pacing at your monitor 's refresh rate?

It looks as though my ini file became a Frankenbuild of the case sensitive/insensitive versions. I can try to repro it if someone thinks there's value to that.

I deleted my ini file and allowed RC1 to create a fresh one. I had to manually modify it with my key mappings (seemed easier than doing that within VPX). 

Your tip on fps was close: it looks like the Frankenbuild mapping of the ini caused it to map 32 (ColorDepth) into RefreshRate . I'm assuming that was the issue, although it's strange that it opted for 60 instead of 32, since my monitor's refresh rate was 120. 

I can now run at 120fps in both GL and non-GL versions, but launching the GL version is forcing VR even though it's set to auto-detect. Non-GL version seems to be fine. I can disable VR Mode until I sort that out.
